spec.bs

```diff
@@ -1560,8 +1561,10 @@ In the meantime, callers of the API should consider the following:
 * Aim for consistency between values of {{SecurePaymentConfirmationRequest/locale}}
     (when provided) and the language of displayable strings.
 
-* If displayable strings contain or might contain [=bidirectional text=], ensure
-    that they will display correctly without the application needing to process
-    the contents of the string value via the insertion of Unicode bidirectional
-    controls. For more information see [How to use Unicode controls for bidi
-    text](https://www.w3.org/International/questions/qa-bidi-unicode-controls).
+* When an implementation is the [=producer=] of displayable strings
```

Comment:
I'm going to be frank, I don't know what this paragraph says.

- Is the 'implementation' referred to here the User Agent's implementation of SPC?
- If so, what does it mean for the User Agent to be a producer? The i18n definition implies that it is where the user agent **creates** natural language string data, which for SPC is only the case for... any user interface elements that are outside the scope of this spec? (E.g., in Chrome, we have text that says e.g. "Use Touch ID to verify and complete your purchase").
- If so, who is the consumer then? Also the User Agent?
- If so, is this just telling the User Agent to insert Unicode bidirectional controls into its own text?

Ultimately, **what exactly** should 'callers of the API' consider, and **why**??